ports/templatetags/url_generate.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+register = template.Library()
+
+
+@register.simple_tag
+def build_url(builder_name, build_id):
+    return "https://build.macports.org/builders/ports-" + builder_name + "-builder/builds/" + str(build_id)
```

Comment:
would be better if the base URL ("https://build.macports.org/builders/ports-") is not embedded here.

Reply:
I am not sure if I get your point, what else do you suggest?

Reply:
Simply having something like `url_prefix = 'https://build.macports.org'` on top.